bin/elasticsearch.bat

```diff
@@ -1,10 +1,46 @@
 @echo off
 
-SETLOCAL
+SETLOCAL enabledelayedexpansion
 TITLE Elasticsearch ${project.version}
 
+SET params='%*'
+
+:loop
+FOR /F "usebackq tokens=1* delims= " %%A IN (!params!) DO (
+    SET current=%%A
+    SET params='%%B'
+	SET silent=N
+	
+	IF "!current!" == "-s" (
+		SET silent=Y
+	)
+	IF "!current!" == "--silent" (
+		SET silent=Y
+	)	
+	
+	IF "!silent!" == "Y" (
+		SET nopauseonerror=Y
+	) ELSE (
```

Comment:
Wondering what would be the effect if you don't rewrite parameters and keep `--silent`  as an option sent to elasticsearch? 

I would probably try only to detect that `-s` or `--silent` is activated without removing it. Could this work?